advocacy_docs/partner_docs/VeritasGuide/03-SolutionSummary.mdx

```diff
@@ -0,0 +1,9 @@
+---
+title: 'Solution Summary'
+description: 'Brief explanation of the solution and its purpose'
+---
+NetBackup for PostgreSQL Agent extends the capabilities of NetBackup to include backup and restore of PostgreSQL databases. If a NetBackup environment is operational within an organization, then users can backup and restore EDB Postgres Advanced and EDB Postgres Extended database servers with the help of NetBackup for PostgreSQL Agent. 
```

Comment:
All instances of "EDB Postgres Advanced" and "EDB Postgres Extended" should end with servers. I agree with Scott on the image used as well, we should reach out to Tim or Scarlett and ask them to produce an image with the EDB logo in it.

Reply:
I copied the image from the Nutanix guide and updated it here.